manticore/platforms/linux.py

```diff
@@ -1545,23 +1554,23 @@ def sys_ARM_NR_set_tls(self, val):
 
     # Signals..
     def sys_kill(self, pid, sig):
-        logger.debug(f"KILL, Ignoring Sending signal {sig} to pid {pid}")
+        logger.warning(f"KILL, Ignoring Sending signal {sig} to pid {pid}")
         return 0
 
     def sys_rt_sigaction(self, signum, act, oldact):
         """Wrapper for sys_sigaction"""
         return self.sys_sigaction(signum, act, oldact)
 
     def sys_sigaction(self, signum, act, oldact):
-        logger.debug(f"SIGACTION, Ignoring changing signal handler for signal {signum}")
+        logger.warning(f"SIGACTION, Ignoring changing signal handler for signal {signum}")
         return 0
 
     def sys_rt_sigprocmask(self, cpu, how, newset, oldset):
         '''Wrapper for sys_sigprocmask'''
         return self.sys_sigprocmask(cpu, how, newset, oldset)
 
     def sys_sigprocmask(self, cpu, how, newset, oldset):
-        logger.debug(f"SIGACTION, Ignoring changing signal mask set cmd:{how}", )
+        logger.warning(f"SIGACTION, Ignoring changing signal mask set cmd:{how}", )
```

Comment:
Since this is `logging`, we should rather use it's internal formatting (i.e. we don't want to format if we discard the log later on). This being said if we change it to `warning` it's rather okay to format it right away?